storm-core/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1990,31 +1995,26 @@ protected void forceDeleteImpl(String path) throws IOException {
     }
 
     /**
-     * Creates a symbolic link to the target
+     * Creates a symbolic link to the target and force the creation if the target already exists
      * @param dir the parent directory of the link
      * @param targetDir the parent directory of the link's target
-     * @param filename the file name of the link
      * @param targetFilename the file name of the links target
+     * @param filename the file name of the link
      * @throws IOException
      */
-    public static void createSymlink(String dir, String targetDir,
-            String filename, String targetFilename) throws IOException {
+    public static void forceCreateSymlink(String dir, String targetDir,
```

Comment:
The sequence of parameters have been swapped: targetFilename and filename.
Since parameters of parent directory are link and target, it would be natural to keep that sequence to filename, too.